chat_shell/.env.example

```
@@ -73,6 +73,8 @@ CHAT_SHELL_MAX_CONCURRENT_CHATS=50
 CHAT_SHELL_CHAT_HISTORY_EXPIRE_SECONDS=7200
 # Maximum messages to keep in history
 CHAT_SHELL_CHAT_HISTORY_MAX_MESSAGES=50
+# Extra attempts for retryable remote history requests (0 disables retries)
+CHAT_SHELL_CHAT_HISTORY_RETRY_COUNT=1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHAT_SHELL_CHAT_HISTORY_RETRY_COUNT key should go before the CHAT_SHELL_MAX_CONCURRENT_CHATS key

(UnorderedKey)